backends/db.go

```diff
@@ -41,5 +42,7 @@ func OpenDatabase(dsn, engine string, tries int) (*sqlx.DB, error) {
 		return nil, fmt.Errorf("couldn't ping database %s: %s", engine, err)
 	}
 
+	db.SetConnMaxLifetime(time.Minute)
```

Comment:
Maybe we should make this an option so users may tune it. What do you think?

Reply:
Hey, that could be a good idea? Unfortunately I'm not a Go expert, and I'm not sure how to implement that :) Do you want to have a look at it?

Reply:
@thomastvedt sorry, I was on vacation and just got back. Sure, I'll do so.

Reply:
Great, thanks 😃

Reply:
Terribly sorry for the delay, but here it is, @thomastvedt: https://github.com/iegomez/mosquitto-go-auth/pull/224